README.md

```diff
@@ -298,7 +298,7 @@ run:
 
 If you have any questions we're in #docker-machine on Freenode.
 
-=======
+=========
```

Comment:
Perhaps I am missing something here, but couldn't we just remove these to prevent the conflicts?